ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -189,6 +196,9 @@ - (UIColor *)mapView:(MGLMapView *)mapView fillColorForPolygonAnnotation:(RCTMGL
 - (BOOL)mapView:(RCTMapboxGL *)mapView annotationCanShowCallout:(id <MGLAnnotation>)annotation {
     NSString *title = [(RCTMGLAnnotation *) annotation title];
     NSString *subtitle = [(RCTMGLAnnotation *) annotation subtitle];
+    if (_annotationsPopUpEnabled == NO)
+        return NO;
+    else
```

Comment:
`else` after return isn't required. Also, you don't need to compare a boolean with NO, you can just
`if (!_annotationsPopUpEnabled) { return NO; }`